PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -1,3 +1,6 @@
+_Last Updated: (date form was last changed)_
+_Last Reviewed: (date form was last reviewed for accuracy)_
```

Comment:
FOLIO uses Jira to track progress on tasks like this. Example, not "Issues in epic": https://issues.folio.org/browse/FOLREL-533

What advantage does this git based progress tracking have? Isn't it duplicating the "Status" field of the Jira issue?

Reply:
As I recall, the reason for adding these lines to the PDD file was for automated reporting. We can write a script that scans the PDD files in FOLIO repositories, then uses the [Team vs module responsibility matrix](https://wiki.folio.org/display/REL/Team+vs+module+responsibility+matrix) to send follow-up messages when a module's PDD hasn't been reviewed in a while.  Modification date we could get from Git; review date would be harder.